src/test/java/io/codeka/gaia/dashboard/controller/UiExtensionsScannerTest.kt

```diff
@@ -0,0 +1,41 @@
+package io.codeka.gaia.dashboard.controller
+
+import io.codeka.gaia.config.security.SecurityConfig
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.context.ApplicationContext
+import org.springframework.data.mongodb.core.MongoTemplate
+
+@SpringBootTest(classes = [UIExtensionsConfig::class])
+class UiExtensionsScannerTest(val applicationContext: ApplicationContext) {
+
+    @Test
+    fun scan() {
+        val scanner = UiExtensionsScanner(applicationContext)
+
+        val locations = arrayOf(
+                // CSS
+                "classpath*:/**/bootstrap/**/bootstrap.min.css",
```

Comment:
what do you think about using a glob pattern gathering JS and CSS to avoid too many declarations and to have only one line by library ?

For example, for bootstrap:
```
/**/bootstrap/**/bootstrap.min.@(css|js)
```

Reply:
I thought about it first.

But using one line by library allows us to be sure that every expression gets its unique file.

I changed the `resolveAsset` function to use the `single()` method on the resource array, to ensure that only _one_ resource file is load for each resource declaration, as it throws an exception if the array is empty or has more that one file. Then preventing the app to load if a resource file cannot be found !

I added a test case to validate this behaviour.